calibr8/__init__.py

```diff
@@ -46,8 +46,7 @@
     HAS_PYMC,
     istensor,
     plot_model,
-    plot_norm_band,
-    plot_t_band,
+    plot_continuous_band,
```

Comment:
Can we keep `plot_norm_band` and `plot_t_band` around for backwards-compatibility? (With a deprecation warning.)
Otherwise it would be a breaking change for a lot of our own code.

Reply:
Yep, sorry, I will add that!